blocks/customers/customers.css

```diff
@@ -0,0 +1,100 @@
+.customers{
+    margin-top: 20px;
+    margin-bottom: 60px;
+    padding:0 10px;
+}
+.customers__image{
```

Comment:
Another way to do this overlay could be to set the customer__content as the content container, set a background image of this image to that container, space text and such using flex-box within that container so as not to have text on an aboslutely positioned element, then use `customers__content:after` to set the overlay style. this would require absolute positioning only on the overlay, and relative positioning with a z-index greater than that of the overlay in order to have the text appear. The way you did it works, so I'm not suggesting this be changed, but just wanted to make a note.